packages/docusaurus-plugin-content-blog/src/index.ts

```diff
@@ -414,5 +420,55 @@ export default function pluginContentBlog(
         }),
       );
     },
+
+    injectHtmlTags() {
+      if (!options.feedOptions) {
+        return;
+      }
+
+      const feedTypes = getFeedTypes(options.feedOptions);
+      const {
+        siteConfig: {baseUrl = '', title},
+      } = context;
+      const feedsConfig: {
+        [key: string]: {type: string; path: string; title: string};
+      } = {
+        rss: {
+          type: 'application/rss+xml',
+          path: 'blog/rss.xml',
+          title: `${title} Blog RSS Feed`,
+        },
+        atom: {
+          type: 'application/atom+xml',
+          path: 'blog/atom.xml',
+          title: `${title} Blog Atom Feed`,
+        },
+      };
+      const headTags: object[] = [];
```

Comment:
headTags should be of type htmlTags 
https://github.com/facebook/docusaurus/blob/61551152805aa53b1a50abf307ed6491ff3ab742/packages/docusaurus-types/src/index.d.ts#L82

Reply:
See https://github.com/facebook/docusaurus/blob/61551152805aa53b1a50abf307ed6491ff3ab742/packages/docusaurus-types/src/index.d.ts#L115-L119